examples/create_ingress.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kubernetes import client, config
+
+
+def main():
+    # Fetching and loading Kubernetes Information
+    config.load_kube_config()
+    # For incluster details
+    # config.incluster_kube_config()
```

Comment:
[`load_incluster_config`](https://github.com/kubernetes-client/python-base/blob/4f7bcfaa91d8f95d358f8215b4efacda734492c2/config/incluster_config.py#L89) (also in the other files)